packages/foam-vscode/src/dated-notes.ts

```diff
@@ -38,6 +59,16 @@ function getDailyNotePath(
   }
 }
 
+/**
+ * Get the daily note filename.
+ *
+ * This function will fetch the filename format and extension from
+ * `foam.openDailyNote.filenameFormat` and `foam.openDailyNote.fileExtension`, respectively.
```

Comment:
```suggestion
 * Get the filename (basename) to use.
 *   
 * It fetch the filename format and extension from
 * `foam.openDailyNote.filenameFormat` and `foam.openDailyNote.fileExtension`, respectively.
```

Describing what the function does. As it stands, the comment describes how it does it.
Following the same format as the others in this PR.